File: src/main/kotlin/week2/section4/ArithmeticOperatorExample.java
package week2.section4;

import java.util.Scanner;

/**
 * 산술연산자의 예제 코드
 */
public class ArithmeticOperatorExample {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("초단위의 정수를 입력하시오> ");
        int input = scanner.nextInt();        // 정수입력

		int hours = input / 60 / 60;        // 시간
		int minutes = input / 60 % 60;      // 나머지 분
		int seconds = input % 60;           // 나머지 초

        System.out.printf("%d초는 %d시간 %d분 %d초 입니다.",
                input, hours, minutes, seconds);
    }
}

File: src/main/kotlin/week2/section4/ArithmeticOperatorExample.kt
package week2.section4

import java.util.*

fun main() {
    val scanner = Scanner(System.`in`)
    print("초단위의 정수를 입력하시오> ")
    val input = scanner.nextInt() // 정수입력

    val hours = input / 60 / 60
    val minutes = input / 60 % 60
    val seconds = input % 60

    print("${input}초는 ${hours}시간 ${minutes}분 ${seconds}초 입니다.")
}